cmd/notary-server/config.go

```diff
@@ -90,7 +91,7 @@ func getStore(configuration *viper.Viper, hRegister healthRegister, doBootstrap
 	switch backend {
 	case notary.MemoryBackend:
 		return storage.NewMemStorage(), nil
-	case notary.MySQLBackend, notary.SQLiteBackend:
+	case notary.MySQLBackend, notary.SQLiteBackend, notary.PostgresBackend:
```

Comment:
Hi @dqminh would you mind also update the relative [configuration docs](https://github.com/docker/notary/blob/master/docs/reference/server-config.md#storage-section-required) about this backend, it's OK to do it in an separated PR, but we may forget to update it.